examples/taproot-psbt.rs

```diff
@@ -0,0 +1,556 @@
+//! Example of taproot PSBT workflow
+
+// We use the alias `alias bt='bitcoin-cli -regtest'` for brevity.
+
+// Step 0 - Wipe the `regtest` data directory to start from a clean slate.
+
+// Step 1 - Run `bitcoind -regtest -daemon` to start the daemon.
+
+// Step 2 - 
+//          2.1) Run `bt -named createwallet wallet_name=benefactor descriptors=true` to create a new descriptor wallet with the name "benefactor"
```

Comment:
I couldn't get this to work with v22 but v23 has been released now so I'd suggest just stating that the example docs are for Bitcoin Core v23. If you do that you can remove the `descriptors=true` from this and below.